cvat-ui/src/components/header/header.tsx

```diff
@@ -245,154 +246,159 @@ function HeaderContainer(props: Props): JSX.Element {
         }
     };
 
-    let upgradeMenuItem = null;
-    if (CVAT_BILLING_URL) {
-        let upgradeText = 'Upgrade to Pro';
-        let upgradeLink = `${CVAT_BILLING_URL}/?type=personal`;
-        if (currentOrganization) {
-            upgradeText = 'Upgrade to Team';
-            upgradeLink = `${CVAT_BILLING_URL}/?type=organization&orgId=${currentOrganization.id}`;
-        }
-        upgradeMenuItem = (
+    const plugins = usePlugins((state: CombinedState) => state.plugins.components.header.userMenu.items, props);
+
+    const menuItems: [JSX.Element, number][] = [];
+    if (user.isStaff) {
+        menuItems.push([(
             <Menu.Item
-                className='cvat-menu-item-highlighted'
-                icon={<UpgradeIcon />}
-                key='upgrade'
-                onClick={() => window.open(upgradeLink, '_self')}
+                icon={<ControlOutlined />}
+                key='admin_page'
+                onClick={(): void => {
+                    // false positive
+                    // eslint-disable-next-line
+                    window.open(`${tool.server.host}/admin`, '_blank');
```

Comment:
Do we really have eslint error here? I don't see any problems if I remove `// eslint-disable-next-line` and run eslint localy

Reply:
Before it raised the error like ``unsafe open file`` or something like this, maybe eslint rule was fixed, maybe something else. 
Need to check, if it is not necessary anymore, we can remove.